scripts/generate-google3-docs/resource-reference/generated/resource-docs/cloudbuild/cloudbuildworkerpool.md

```diff
@@ -0,0 +1,592 @@
+{# AUTOGENERATED. DO NOT EDIT. #}
+
+{% extends "config-connector/_base.html" %}
+
+{% block page_title %}CloudBuildWorkerPool{% endblock %}
+{% block body %}
+
+
+<table>
+<thead>
+<tr>
+<th><strong>Property</strong></th>
+<th><strong>Value</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>{{gcp_name_short}} Service Name</td>
+<td>Cloud Build</td>
+</tr>
+<tr>
+<td>{{gcp_name_short}} Service Documentation</td>
+<td><a href="/cloud-build/docs/">/cloud-build/docs/</a></td>
+</tr>
+<tr>
+<td>{{gcp_name_short}} REST Resource Name</td>
+<td>v1.projects.workerpools</td>
+</tr>
+<tr>
+<td>{{gcp_name_short}} REST Resource Documentation</td>
+<td><a href="/cloud-build/docs/api/reference/rest/v1/projects.locations.workerPools">/cloud-build/docs/reference/rest/v1/projects.locations.workerPools</a></td>
+</tr>
+<tr>
+<td>{{product_name_short}} Resource Short Names</td>
+<td>cloudbuildworkerpool</td>
+</tr>
+<tr>
+<td>{{product_name_short}} Service Name</td>
+<td>cloudbuild.googleapis.com</td>
+</tr>
+<tr>
+<td>{{product_name_short}} Resource Fully Qualified Name</td>
+<td>cloudbuildworkerpools.cloudbuild.cnrm.cloud.google.com</td>
+</tr>
+
+<tr>
+    <td>Can Be Referenced by IAMPolicy/IAMPolicyMember</td>
+    <td>No</td>
+</tr>
+
+
+</tbody>
+</table>
+
+## Custom Resource Definition Properties
+
+
+### Annotations
+<table class="properties responsive">
+<thead>
+    <tr>
+        <th colspan="2">Fields</th>
+    </tr>
+</thead>
+<tbody>
+    <tr>
+        <td><code>cnrm.cloud.google.com/state-into-spec</code></td>
+    </tr>
+</tbody>
+</table>
```

Comment:
@maqiuyujoyce Do you have any suggestions how we can remove this state-into-spec annotation from the g3doc? I think this is auto-generated (I removed the sts annotation in the tmpl).

Reply:
A quick workaround would be add a condition check here: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/02d6130fd03fe40ccb0be8e3953092bc947debad/scripts/generate-google3-docs/resource-reference/main.go#L345 (e.g. if rc.Direct == true then skip populating the annotation section).

If we want to support other annotations, we might add them incrementally with more graceful code changes in the future.

Reply:
Uploaded https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2186.